src/test/java/uk/gov/hmcts/reform/sscs/service/ccdupdate/CcdLocationUpdateServiceTest.java

```diff
@@ -0,0 +1,178 @@
+package uk.gov.hmcts.reform.sscs.service.ccdupdate;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Address;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Hearing;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Venue;
+import uk.gov.hmcts.reform.sscs.exception.UpdateCaseException;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+import uk.gov.hmcts.reform.sscs.model.messaging.HearingUpdate;
+import uk.gov.hmcts.reform.sscs.model.messaging.HmcMessage;
+import uk.gov.hmcts.reform.sscs.service.venue.VenueRpcDetailsService;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CcdLocationUpdateServiceTest {
+
+    public static final String HEARING_ID = "789";
+    public static final String EPIMS_ID = "123";
+    public static final String NEW_EPIMS_ID = "456";
+    public static final String CASE_ID = "777";
+    public static final String VENUE_NAME = "VenueName";
+
+    @Mock
+    private VenueRpcDetailsService venueRpcDetailsService;
+
+    @InjectMocks
+    private CcdLocationUpdateService underTest;
+
+    @Test
+    void testShouldFindVenueByVenueId() {
```

Comment:
this test seems a bit pointless the way it is currently.. you don't need to mock the venueDetails object and assert it - you're just asserting your own test data then - the only thing you need to test is the venueId lookup works - consider stripping this test back to reflect this i.e. just return an empty venueDetails and make sure it's not null.

update: ahh.. I looked at the method and I see why you're doing this now you're mapping VenueDetails to Venue!